src/panels/config/integrations/integration-panels/zha/dialog-zha-reconfigure-device.ts

```diff
@@ -289,6 +291,12 @@ class DialogZHAReconfigureDevice extends LitElement {
                                               `
                                             : html`
                                                 <span class="stage">
+                                                  <simple-tooltip
+                                                    animation-delay="0"
+                                                    position="top"
+                                                  >
+                                                    ${attribute.status}
```

Comment:
We should add translations for this

Reply:
Sure, I can do that in a followup PR. They haven't existed up to this point and the backend for this was already merged.

Reply:
Not sure what the best way for this is, but showing the "raw error code / status" might still be good.
I think you'd find more results when searching for something like `UNREPORTABLE_ATTRIBUTE` compared to the translation, because that's exactly how it's called in ZCL. So, also the same error code in Z2M/deCONZ probably.

![image](https://github.com/home-assistant/frontend/assets/6409465/756cf027-98b4-4e92-8a26-04afb293d2fd)
(But I guess this is more of a discussion for a follow-up PR.)